lib/groonga/command/parser/command/groonga-command-convert-format.rb

```diff
@@ -95,8 +96,23 @@ def parse_options!(argv)
 
           def convert(input)
             parser = Parser.new
-            parser.on_command do |command|
-              puts(convert_format(command))
+            case @format
+            when :elasticsearch
+              loaded_values = []
+              parser.on_load_value do |command, value|
+                loaded_values << value
+              end
+              parser.on_load_columns do |command, header|
+                loaded_values << header
+              end
+              parser.on_load_complete do |command|
+                command[:values] = loaded_values.to_s
```

Comment:
Is `to_s` correct?
I think that `to_json` or `JSON.generate(loaded_values)` is correct.